assembl/static/js/app/views/ideaPanel.js

```diff
@@ -737,6 +741,7 @@ var IdeaPanel = AssemblPanel.extend({
                   that.model.destroy({
                     success: function() {
                       that.unblockPanel();
+                      // UX question: should we go to the parent idea, if any?
```

Comment:
Good we agree. Out of scope, of course, just adding a TODO for the future because I saw it.